federation-js/src/composition/validate/preComposition/__tests__/duplicateEnumOrScalar.test.ts

```diff
@@ -89,6 +97,12 @@ describe('duplicateEnumOrScalar', () => {
       Array [
         Object {
           "code": "DUPLICATE_SCALAR_DEFINITION",
+          "locations": Array [
```

Comment:
Repeat from above: can we capture all locations of the duped scalar?

Reply:
^ same as above